quickstart/deploy-a-test-application.md

```diff
@@ -44,13 +44,21 @@ spec:
   storageClassName: INSERT_YOUR_STORAGECLASS_NAME_HERE
 ```
 {% endtab %}
+{% tab title="Sample Output" %}
+```text
+NAME              STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+ms-volume-claim   Bound    pvc-ebdfaa3f-5135-4266-b8ad-c8ac553c87fd   1Gi        RWO            mayastor-1     7s
+```
+{% endtab %}
 {% endtabs %}
 
-If you used the storage class example from previous stage, then volume binding mode is set to `WaitForFirstConsumer`. That means, that the volume won't be created until there is an application using the volume. We will go ahead and create the application pod and then check all resources that should have been created as part of that in kubernetes.
+{% hint style="note" %}
+If you used a storage class with the volume binding mode set to `WaitForFirstConsumer` the volume won't be created until there is an application using the volume.
+{% endhint %}
 
 ## Deploy the FIO Test Pod
 
-We schedule the application to one of the storage nodes in order to make the volume available to the application in spite of the node failures. That's why nodeSelector is used in the Pod specification. See ["local" storage class parameter description](https://mayastor.gitbook.io/introduction/reference/storage-class-parameters#local) for more in-depth explanation of how the scheduling works.
+We schedule the application to one of the storage nodes in order to make the volume available to the application in spite of the node failures. That's why `nodeSelector` is used in the Pod specification. See ["local" storage class parameter description](https://mayastor.gitbook.io/introduction/reference/storage-class-parameters#local) for more in-depth explanation of how the scheduling works.
```

Comment:
Should we be talking about "local" parameter or is that defaulted as true? I thought it was the latter.